### PR TITLE
Always check connectivity to overlay network

### PR DESCRIPTION
**Pull request by michalskalski**
When kube-apiserver is no longer able connect overlay network
where oAuth PODs resides it still remains active, but cannot
properly handle requests. In that case kube-apiserver should
be marked as not ready by one of the readyz checks.
Current checks only verify connectivy once after
kube-apiserver started. Modify checks to constantly monitor
connection to overlay network.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: openshift-kube-apiserver/openshiftkubeapiserver/sdn_readyz_wait.go [hunk 1]
```diff
@@ -93,54 +73,28 @@ func (c *aggregatedAPIServiceAvailabilityCheck) checkForConnection(context gener
 			},
 			Timeout: 1 * time.Second, // these should all be very fast.  if none work, we continue anyway.
```
Comment (aojea) [inline openshift-kube-apiserver/openshiftkubeapiserver/sdn_readyz_wait.go:74]: 1 second is tooo short in production environments

Comment (michalskalski) [reply openshift-kube-apiserver/openshiftkubeapiserver/sdn_readyz_wait.go:74]: Hi, part of the code which do a http request is not changed, the 1 second value is used in current releases, should that be changed in your opinion?

Comment (aojea) [reply openshift-kube-apiserver/openshiftkubeapiserver/sdn_readyz_wait.go:74]: yeah, I just noticed now, forget the comment, I really understand it well

**Comment by michalskalski** [comment]
Initial issue was solved by extending list of resources for which connectivity to aggregated api is not required https://github.com/openshift/library-go/commit/5add9e2f611e27572fc064a05a4d6331659c1e76

**Pull request closed without merging**